docs/supported-technologies.asciidoc

```diff
@@ -0,0 +1,84 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/dotnet[elastic.co]
+endif::[]
+
+[[supported-technologies-details]]
```

Comment:
I recommend changing this attribute to simply "supported-technologies". This will fix the doc build.

```suggestion
[[supported-technologies]]
```